app/models/enterprise.rb

```diff
@@ -419,16 +389,16 @@ def ensure_owner_is_manager
     users << owner unless users.include?(owner) || owner.admin?
   end
 
-  def ensure_email_set
-    self.email = owner.email if email.blank? && owner.present?
-  end
-
   def enforce_ownership_limit
     unless owner.can_own_more_enterprises?
       errors.add(:owner, I18n.t(:enterprise_owner_error, email: owner.email, enterprise_limit: owner.enterprise_limit ))
     end
   end
 
+  def set_default_contact
+    EnterpriseRole.set_notification_user self.owner_id, self.id
```

Comment:
As per (this comment)[https://github.com/openfoodfoundation/openfoodnetwork/pull/1720/files#r148708156]: this could become: 

````ruby
def set_default_contact
  set_contact(owner_id)
end